src/arizona_parse_transform.erl

```diff
@@ -65,13 +111,52 @@ it was found, used for error reporting during parse transform processing.
 """.
 -type template_content() :: {binary(), pos_integer()}.
 
+-doc ~"""
+Function specification for arizona_parse_transform attribute.
+
+Tuple containing function name and arity for functions that should
+be analyzed for variable bindings.
+""".
+-type function_spec() :: {FunctionName :: atom(), Arity :: non_neg_integer()}.
+
+-doc ~"""
+Enhanced stateful result with vars_indexes for efficient change detection.
+
+Extends the basic stateful_result from arizona_parser with vars_indexes
```

Comment:
If you use stuff like `arizona_parser` you get links for free, I'm pretty sure.

Reply:
Hm, I didn't get your comment. Can you please provide an example?
Something like this?

```erlang
-type enhanced_stateful_result() :: arizona_parser:stateful_result() | #{
    vars_indexes := #{BindingName :: atom() => [ElementIndex :: non_neg_integer()]}
}.
```

Reply:
Yeah, the UI rendering doesn't help.

I was stating using 

```
Extends the basic stateful_result from `arizona_parser` with vars_indexes
                                       ^ ticks
```

Reply:
Ah, got it! We can improve those things later. I need to do a complete review of the docs.